sass/woocommerce/_archives.scss

```diff
@@ -83,7 +83,7 @@
 				text-align: center;
 
 				@media (max-width: 768px) {
-					& {
+					@at-root body.responsive#{&} {
```

Comment:
You can also check out http://sass-lang.com/documentation/file.SASS_REFERENCE.html#at-root